<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ayeesha" ignored="0" name="Test" tests="1" failures="1" timestamp="2020-03-26T17:08:11 IST" time="10.966" errors="0">
  <testcase name="tc_001_Login" time="10.915" classname="com.learning.testcases.TC_001_Login">
    <failure type="java.lang.AssertionError" message="Login not successful expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login not successful expected [true] but found [false]
at com.learning.testcases.TC_001_Login.tc_001_Login(TC_001_Login.java:16)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- tc_001_Login -->
</testsuite> <!-- Test -->
